integ/validate_cloudwatch/validator.py

```diff
@@ -5,59 +5,80 @@
 import time
 from datetime import datetime, timedelta
 
-
 client = boto3.client('logs', region_name=os.environ.get('AWS_REGION'))
 metrics_client = boto3.client("cloudwatch", region_name=os.environ["AWS_REGION"])
-start_time = datetime.utcnow() - timedelta(seconds=600)
+start_time = datetime.utcnow() - timedelta(seconds=1200)
 end_time = datetime.utcnow()
 
 LOG_GROUP_NAME = os.environ.get('LOG_GROUP_NAME')
 
+
+def execute_with_retry(retriable_function, *argv):
+    max_attempts = 20
```

Comment:
I think max_attempts should be an arg? We don't want to retry the main CW tests this many times? This many retries is just for the EMF tests where the metric takes a while to populate in CW...

Reply:
We can hard code the different retries for different test types in the script

Reply:
Did the changes!